modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -97,7 +97,7 @@ definition {
 			AccessibilityMenu.openAccessibilityMenu();
 
 			Uncheck.uncheckNotVisible(
-				key_toggleSwitchLabel = "Expanded Text",
+				key_toggleSwitchLabel = "Underlined Links",
```

Comment:
Be sure to update the task description to match

Reply:
cc/ @eltonccdantas

Reply:
Alright... done!